drf_jsonapi/mixins.py

```diff
@@ -142,7 +145,8 @@ def create(self, request):
             data=request.data['data'],
             only_fields=request.fields,
             include=request.include,
-            page_size=self.request.GET.get('page[size]', settings.DEFAULT_PAGE_SIZE)
+            page_size=self.request.GET.get('page[size]', settings.DEFAULT_PAGE_SIZE),
```

Comment:
As above, why are we referencing `self.request` when we're passing `request` into this function as an argument?

Reply:
I'll update these as well.